example/provider-local/garden/base/kustomization.yaml

```diff
@@ -1,10 +1,14 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
+sortOptions:
+  order: fifo
```

Comment:
Why is this (suddenly) needed?

Reply:
Kustomize by default applies resources in some legacy order that uses groups + kids. This means that `WorkloadIdentity` is after `CredentialsBinding`. I want kustomize to honour the order from the [credentialsbinding file](https://github.com/gardener/gardener/blob/0f11fd7cd17542cb6f258cd87447c4b02ed50552/example/provider-local/garden/base/credentialsbinding.yaml). If the order is not honoured then validation will fail because the referenced `WorkloadIdentity` does not exist.